unit-test/background/classes/tab.js

```diff
@@ -15,7 +14,7 @@ let tab
 describe('Tab', () => {
     describe('updateSite()', () => {
         beforeEach(() => {
-            spyOn(browserWrapper, 'getExtensionId').and.returnValue('sdf')
+            // spyOn(browserWrapper, 'getExtensionId').and.returnValue('sdf')
```

Comment:
Is this a typo? Or can we just remove this?

Reply:
Not a typo - commenting it out didn't affect the test. I'll delete the line fully.